src/test_workflow/integ_test/distribution_zip.py

```diff
@@ -33,8 +33,8 @@ def install(self, bundle_name: str) -> None:
     @property
     def start_cmd(self) -> str:
         start_cmd_map = {
-            "opensearch": "./opensearch-windows-install.bat",
-            "opensearch-dashboards": "./opensearch-dashboards.bat",
+            "opensearch": ".\\opensearch-windows-install.bat",
+            "opensearch-dashboards": ".\\opensearch-dashboards.bat",
```

Comment:
Nit: just realized that `DistributionZip` is really a `DistributionWindowsZip` and `DistributionTar` is really `DistributionLinuxTar`. Consider renaming.